### DISCUSSION TITLE
Guess the song 🎸

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1095] [Galaxy Trivia] [answered]
```
e|-----15----------------------------------------------|---------18-----18-----18----18-------15----|
B|--------15---18b18b18b-18b-18-15-18-18r-15-18b-18-15-|--18b18b----18b----18b----18b---18b------15-|
G|--15-------------------------------------------------|-----------------------------------15-------|
D|-----------------------------------------------------|--------------------------------------------|
A|-----------------------------------------------------|--------------------------------------------|
E|-----------------------------------------------------|--------------------------------------------|
```

**Bonus:** Name the songwriter.

**Comment by peterbaccaro** [comment]
The song is **Free Bird** (also spelled Freebird) by American rock band **Lynyrd Skynyrd**, written by guitarist **Allen Collins** and lead singer **Ronnie Van Zant**.

**Comment by sheri-na** [comment]
The song is called Freeboard by the band Lynyrd Skynyrd but the songwriters are Allen Collins and Ronnie Van Zant.

**Comment by thuongtruong1009** [comment, marked as answer]
The song you provided tabs for is "Sultans of Swing" by Dire Straits. The songwriter is Mark Knopfler, who is also the lead vocalist and guitarist of the band.

**Reply by peterbaccaro** [reply]
@ParanoidUser, can you share the tab that shows that Sultans of Swing by Dire Straits is the correct answer?

Here is a link to the tab that shows that Free Bird by Lynyrd Skynyrd is a match...

https://tabs.ultimate-guitar.com/tab/lynyrd-skynyrd/free-bird-tabs-350756

![image](https://github.com/ParanoidUser/yolo/assets/74666079/dba912c6-195c-4b9b-bedc-a1a8df2a30e5)

**Reply by ParanoidUser** [reply]
Woop-woop, woop-woop! 🚨🚔

📢 Attention all personnel, this is the Musical Intelligence Division. I repeat, the Musical Intelligence Division has arrived on the scene. Maintain your current positions. The melody mayhem ends now! 🎵👮

---

#### Incident Report – Case #84723
**Date:** August 14, 2023
**Reporting Officer:** Detective Paranoid Muser
**Subject:** Incorrect Song Identification

**Description of Incident:**
On August 14, 2023, the Musical Intelligence Division was alerted to a case of song identity confusion. The song in question, "Freeboard" by Lynyrd Skynyrd, was incorrectly replaced by "Sultans of Swing" by Dire Straits in a recent musical knowledge repository update. Our vigilant observer, @peterbaccaro, astutely detected this substitution and hinted at a subtle.

**Evidence:**
 - Timestamped Aug 11 alteration to "Sultans of Swing" answer.
 - Forensic analysis of iconic party popper 🎉 emojis.

**Action Taken:**
In accordance with the harmonic harmony act of 1970, Section 42(b)(2), impose a fine on @thuongtruong1009 for their audacious audio antics. The sound waves shall be fined in the form of an irresistible dance-off challenge.

Furthermore, an immediate rollback to the original and unmistakably righteous "Freebird" response is hereby enacted.
Let the harmonious equilibrium of the musical cosmos be restored!

🎵🎶 *Case closed, until the next riff. Keep groovin' responsibly!* 🎵🎶
